packages/core/core/src/index.js

```diff
@@ -1,5 +1,12 @@
 export { ProcessLauncher } from './processes/processLauncher';
-export { ProcessManager } from './processes/processManager';
 export { ProcessWrapper } from './processes/processWrapper';
 
+export { Config } from './config';
 export { IPC } from './ipc';
+export { Engine } from './engine';
+export { EmbarkEmitter as Events } from './events';
+export { Plugins } from './plugins';
+export { TestLogger } from './utils/test_logger';
+export { TemplateGenerator } from './utils/template_generator';
```

Comment:
There are a few symbols that should not be exported here, but are for the time being (didn't want to make this PR more complex as it just focusses on moving `Engine` and alike to `embark-core`:

- `TestLogger` should probably go into `embark-testing` at some point (if we really need it, otherwise we can drop it)
- `TemplateGenerator` is potentially something we can either move somewhere else, or have `Engine` expose an API that makes use of `TemplateGenerator` so that consumers can just import and use `Engine` (and don't have to worry about `TemplateGenerator`).